ModProg/derive-where
# Attributes on fields and variants

- Start implementing tests
- Fix and test `union` error message
- Convert errors instead of taking a reference
- Improve error message for missing or wrong trait
- Make tests actually able to build
- Add test for unsupported item without generics
- Adjust to crate rename
- Test unit structs
- Test output directly
- Improve `PartialEq` implementation for `enum`s
- Invert syntax and test all error messages
- Run rustfmt
- Adjust MSRV, enable MSRV testing and reduce dependencies
- Address review comments
- Improve `PartialEq` implementation for `enum`s
- Add tests for edge-cases
- Improve `Ord` and `PartialOrd` implementation to be closer to `std`s derive
- Improve structure to prevent duplicate code
- Only convert to string when absolutely necessary
- Implement deriving for a union
- Add `#[inline]` to signatures according to the standard derive implementation
- Do some spellchecking
- Implement safe alternative
- Implement unsafe version of `PartialOrd` and `Ord`
- Add another test to make sure everything remains the same
- Fix `Ord` and `PartialOrd` implementation
- Restore MSRV
- Remove unnecessary unit value
- Implement `nightly` crate feature
- Improve warning circumvention
- Add `resolver = "2"` because we aren't on Rust edition 2021 anymore
- Implement `Zeroize` trait
- Split MSRV compatible tests into separate crate and implement `Zeroize` `drop` and `crate` support
- Code de-duplication
- Improve `PartialEq`
- Improve empty struct or tuple variant and add tests
- Start implementing integration tests
- Support derive_where as attribute
